test/integration/signup_flow_test.rb

```diff
@@ -0,0 +1,103 @@
+require 'test_helper'
+
+
+class SignUpTest < ActionDispatch::IntegrationTest
+  def setup
```

Comment:
can we use already created usernames and passwords like :bob, :jeff etc

Reply:
If we used already created usernames and emails, it would fail to create the user because the username already exists.

Reply:
Oh got it!